user-research/contributor-research/Introduction.md

```diff
@@ -0,0 +1,31 @@
+This research aims to understand why some individuals sustain active involvement in Knative while others contribute sporadically or disengage entirely. The ultimate goal is to identify aspects of the contributor experience that could be improved to enhance contributor retention and propose actionable recommendations to tackle these issues.
+
+## Research Goals:
+- Understand the current contributor journey within Knative.
+- Identify factors that contribute to contributors staying or leaving the Knative community.
+- Highlight areas of the contributor experience that can be improved to increase contributor retention and propose ways to improve those areas.
+
+## Objectives:
+- Gather insights into the onboarding process for new contributors to Knative.
+- Explore the experiences of both retained and departed contributors.
+- Identify pain points and areas of dissatisfaction throughout the contributor journey.
+- Investigate the effectiveness of existing resources and documentation for contributors.
+- Determine potential interventions or improvements to enhance the contributor experience and retention rate.
+
+## Key Research Questions:
+- What motivated contributors to join the Knative community initially and what were their expectations regarding their involvement in the projects?
+- What were the primary challenges they encountered during the onboarding process? (e.g. selecting the right project and right issues to contribute to as per one’s technical expertise)
+- How did contributors interact with the Knative community, and what were their experiences regarding communication and collaboration within the community
+- At what stage did certain contributors typically begin to disengage from the community?
+- What factors lead contributors to discontinue their engagement with Knative, and what practical recommendations do they offer to enhance the contributor experience, fostering a more sustainable and active ecosystem?
+- Are there any differences in experiences between contributors with varying levels of technical expertise or backgrounds?
+- What is missing in the current contributor experience of Knative as compared to some other well-established open-source projects?
+
+## Hypotheses:
+- Contributors who have a positive onboarding experience, are proactive, and receive clear guidance and support, are more likely to stay engaged with the Knative community.
+- Factors such as perceived value of contributions, prospective future opportunities, and alignment with personal or professional goals influence contributors' decisions to remain active in the project.
+- Contributors may disengage due to challenges like unclear contribution processes, technical knowledge/expertise barriers, insufficient/delayed feedback, lack of recognition, community integration issues, etc. Additionally, heavy workloads, limited free time, personal commitments, or simply a change of plans may also contribute to disengagement.
+
+
```

Comment:
Consider adding a section on the potential impact of your research findings and how you envision the implementation of your recommendations within the Knative community could be cool? This not only shows the practical relevance of your research but also can help in convincing the community.

Reply:
covered this point in the 'Abstract' section.